<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMeshRendererScalarSettingsWidgetBase</class>
 <widget class="QWidget" name="QgsMeshRendererScalarSettingsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mOpacityContainerWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="mOpacityLabel">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsOpacityWidget" name="mOpacityWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>14</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mEdgeWidthGroupBox">
     <property name="title">
      <string>Stroke width</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QRadioButton" name="mScalarEdgeStrokeWidthVariableRadioButton">
        <property name="text">
         <string>Variable</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="mScalarEdgeStrokeWidthFixedRadioButton">
        <property name="text">
         <string>Fixed</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="3">
       <widget class="QgsMeshVariableStrokeWidthButton" name="mScalarEdgeStrokeWidthVariablePushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="QgsDoubleSpinBox" name="mScalarEdgeStrokeWidthSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="3">
       <widget class="QgsUnitSelectionWidget" name="mScalarEdgeStrokeWidthUnitSelectionWidget" native="true"/>
      </item>
      <item row="0" column="1" rowspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="mScalarMinLabel">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsDoubleSpinBox" name="mScalarMinSpinBox">
       <property name="minimum">
        <double>-10000000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mScalarMaxLabel">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsDoubleSpinBox" name="mScalarMaxSpinBox">
       <property name="minimum">
        <double>-10000000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mScalarRecalculateMinMaxButton">
       <property name="toolTip">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="../../../images/images.qrc">
         <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="mUserDefinedRadioButton">
       <property name="text">
        <string>Use&amp;r defined</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">mMinMaxTypeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mMinMaxRadioButton">
       <property name="text">
        <string>&amp;Min / max</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">mMinMaxTypeButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="mMinMaxValueTypeLabel">
       <property name="text">
        <string>Statistics extent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mMinMaxValueTypeComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="mScalarResamplingWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mLabelDataInterpolation">
        <property name="text">
         <string>Resampling method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mScalarInterpolationTypeComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsColorRampShaderWidget" name="mScalarColorRampShaderWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsUnitSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsunitselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsOpacityWidget</class>
   <extends>QWidget</extends>
   <header>qgsopacitywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorRampShaderWidget</class>
   <extends>QWidget</extends>
   <header>qgscolorrampshaderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMeshVariableStrokeWidthButton</class>
   <extends>QPushButton</extends>
   <header>qgsmeshvariablestrokewidthwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mOpacityWidget</tabstop>
  <tabstop>mScalarEdgeStrokeWidthFixedRadioButton</tabstop>
  <tabstop>mScalarEdgeStrokeWidthVariableRadioButton</tabstop>
  <tabstop>mScalarEdgeStrokeWidthSpinBox</tabstop>
  <tabstop>mScalarEdgeStrokeWidthVariablePushButton</tabstop>
  <tabstop>mScalarEdgeStrokeWidthUnitSelectionWidget</tabstop>
  <tabstop>mScalarMinSpinBox</tabstop>
  <tabstop>mScalarMaxSpinBox</tabstop>
  <tabstop>mScalarRecalculateMinMaxButton</tabstop>
  <tabstop>mScalarInterpolationTypeComboBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="mMinMaxTypeButtonGroup"/>
 </buttongroups>
</ui>
